source/docs/setup.md

```diff
@@ -39,18 +39,19 @@ Application data. The [EJS](https://ejs.co/), [Stylus](http://learnboost.github.
   "version": "0.0.0",
   "private": true,
   "hexo": {
-    "version": ""
+    "version": "7.0.0"
```

Comment:
I guess this is not necessary, hexo will populate this field

https://github.com/hexojs/hexo-starter/blob/a1c722e1dff531cb9ab2ff572eaf4125d3ce150c/package.json#L12